js_modules/dagster-ui/packages/ui-core/src/asset-graph/sidebar/AssetSidebarNode.tsx

```diff
@@ -93,15 +92,27 @@ export const AssetSidebarNode = (props: AssetSidebarNodeProps) => {
   );
 };
 
-const AssetSidebarAssetLabel = ({
+type AssetSidebarAssetLabelProps = {
+  fullAssetGraphData?: GraphData;
+  showStatus?: boolean;
+  node: AssetNodeMenuProps['node'] & StatusDotNode;
+  selectNode: (e: React.MouseEvent<any> | React.KeyboardEvent<any>, nodeId: string) => void;
+  isLastSelected: boolean;
+  isSelected: boolean;
+  explorerPath: ExplorerPath;
+  onChangeExplorerPath: (path: ExplorerPath, mode: 'replace' | 'push') => void;
+};
+
+export const AssetSidebarAssetLabel = ({
```

Comment:
I can't think of a better name for this, but the entire appearance of this (hover state,  ••• on hover, dropdown menu trigger, etc) appear in the new asset health overview.  `AssetPill` or `AssetListButton` or something like that maybe?

Reply:
I like Label better than Pill and Button 🤷🏾